### ISSUE TITLE
New release coming (compatibility with tree-sitter >0.21)?

### ISSUE THREAD

**Issue by jclavier44** [euclidianAce/tree-sitter-teal#29]
Hello thanks for your contribution for this tree-sitter Teal grammar. I really need to extract symbols from a Teal AST in order to be able to use Teal in my project.
But I haven't been able to use your grammar with any latest tree-sitter version.
I have tried last release version 0.0.4 but it seems that the Language definition is not compatible with latest tree-sitter definition. I see that the current main channel code is much newer, do you plan any new release soon?

**Comment by euclidianAce** [comment]
Yes I'll tag a new release soon, especially since teal had a new release recently. This repo hasn't been my highest priority since teal hasn't had many grammar changes itself.

Thanks for your patience.

**Comment by euclidianAce** [comment]
Apologies for the long wait. I've just tagged a new release
